src/Mod/CAM/CAMTests/TestRefactoredTestPostGCodes.py

```diff
@@ -1408,3 +1409,667 @@ def test10980(self):
+            "G1 X0.3937 Y0.7874 Z1.1811 A240.0000 B250.0000 C260.0000",
+            "--inches",
+        )
+
+    #############################################################################
+
+    def test19760(self):
+        """Test combined A, B, & C axis output for values greater than 360 degrees"""
+        self.compare_third_line(
+            "G1 X10 Y20 Z30 A440 B450 C460",
+            "G1 X10.000 Y20.000 Z30.000 A440.000 B450.000 C460.000",
+            "",
+        )
+        self.compare_third_line(
+            "G1 X10 Y20 Z30 A440 B450 C460",
+            "G1 X0.3937 Y0.7874 Z1.1811 A440.0000 B450.0000 C460.0000",
+            "--inches",
+        )
+
+    #############################################################################
+
+    def test19770(self):
+        """Test combined A, B, & C axis output for values between 0 and -90 degrees"""
+        self.compare_third_line(
+            "G1 X10 Y20 Z30 A-40 B-50 C-60",
+            "G1 X10.000 Y20.000 Z30.000 A-40.000 B-50.000 C-60.000",
+            "",
+        )
+        self.compare_third_line(
+            "G1 X10 Y20 Z30 A-40 B-50 C-60",
+            "G1 X0.3937 Y0.7874 Z1.1811 A-40.0000 B-50.0000 C-60.0000",
+            "--inches",
+        )
+
+    #############################################################################
+
+    def test19780(self):
+        """Test combined A, B, & C axis output for values between -90 and -180 degrees"""
+        self.compare_third_line(
+            "G1 X10 Y20 Z30 A-100 B-110 C-120",
+            "G1 X10.000 Y20.000 Z30.000 A-100.000 B-110.000 C-120.000",
+            "",
+        )
+        self.compare_third_line(
+            "G1 X10 Y20 Z30 A-100 B-110 C-120",
+            "G1 X0.3937 Y0.7874 Z1.1811 A-100.0000 B-110.0000 C-120.0000",
+            "--inches",
+        )
+
+    #############################################################################
+
+    def test19790(self):
+        """Test combined A, B, & C axis output for values between -180 and -360 degrees"""
+        self.compare_third_line(
+            "G1 X10 Y20 Z30 A-240 B-250 C-260",
+            "G1 X10.000 Y20.000 Z30.000 A-240.000 B-250.000 C-260.000",
+            "",
+        )
+        self.compare_third_line(
+            "G1 X10 Y20 Z30 A-240 B-250 C-260",
+            "G1 X0.3937 Y0.7874 Z1.1811 A-240.0000 B-250.0000 C-260.0000",
+            "--inches",
+        )
+
+    #############################################################################
+
+    def test19800(self):
+        """Test combined A, B, & C axis output for values below -360 degrees"""
+        self.compare_third_line(
+            "G1 X10 Y20 Z30 A-440 B-450 C-460",
+            "G1 X10.000 Y20.000 Z30.000 A-440.000 B-450.000 C-460.000",
+            "",
+        )
+        self.compare_third_line(
+            "G1 X10 Y20 Z30 A-440 B-450 C-460",
+            "G1 X0.3937 Y0.7874 Z1.1811 A-440.0000 B-450.0000 C-460.0000",
+            "--inches",
+        )
```

Comment:
These 44 tests are nearly identical. To reduce future maintenance effort I suggest to consolidate them into a single, or  perhaps three (normalized, negative and multi-turn angles), tests.

To still keep the coverage up, and to avoid the act-assert-act-assert-act-assert anti-pattern, you could leverage a local function with a [sub-test](https://docs.python.org/3/library/unittest.html#unittest.TestCase.subTest) to get parametrizable tests. [See here](https://github.com/FreeCAD/FreeCAD/blob/a15134d79b29af4e5eb819623b30dcd5c94380d1/src/Mod/CAM/CAMTests/TestPathHelix.py#L142-L167) for an example.

Reply:
Thanks for the feedback and the example.  I agree and will rewrite the tests to make them more maintainable.

Reply:
The rewritten tests are now ready for review and/or pulling.